server/views/components/chapter-indicator/index.config.js

```diff
@@ -0,0 +1,26 @@
+export const name = 'chapter indicator';
+export const handle = 'chapter-indicator';
+export const display = {background: '#1d1d1d'};
+export const variants = [
+  {
+    name: 'default',
+    label: 'Purple',
+    context: {model: {position: 2, series: {items: ['one', 'two', 'three'], total: 5, color: 'purple'}}}
```

Comment:
I'm a little lost as to what these are models of - nice to get some type references in there.